src/adr/0018-replace-jcloud-with-s3.md

```diff
@@ -0,0 +1,41 @@
+# 18. Replace JCloud-based ObjectStorage backend with AWS-S3 SDK backend 
+
+Date: 2020-02-11
+
+## Status
+
+Proposed
+
+Adoption needs to be backed by some performance tests.
+
+## Context
+
+James has an interface called blob-store that allows to choose how Blobs are stored between several implementations.
+
+For Distributed product, the default backend is Cassandra because it's fast and it doesn't require yet another
+server. 
+
+When storage cost concerns are more important than raw performances, James can store Blobs in S3 or Swift 
```

Comment:
Cassandra is also not good at storing large blobs as a recall.

Reply:
I don't share this opinion. AFAIK, it can work very well.